src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -235,25 +290,56 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+            ->end();
+    }
 
+    private function addMercureSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
                 ->arrayNode('mercure')
                     ->{class_exists(MercureBundle::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                     ->children()
                         ->scalarNode('hub_url')
                             ->defaultNull()
-                            ->info('The URL send in the Link HTTP header. If not set, will default to the URL for the Symfony\'s bundle default hub.')
+                            ->info('The URL sent in the Link HTTP header. If not set, will default to the URL for MercureBundle\'s default hub.')
                         ->end()
                     ->end()
                 ->end()
+            ->end();
+    }
 
+    private function addMessengerSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
                 ->arrayNode('messenger')
-                    ->{interface_exists(MessageBusInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                    ->{!class_exists(FullStack::class) && interface_exists(MessageBusInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
                 ->end()
+            ->end();
+    }
 
+    private function addElasticsearchSection(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
                 ->arrayNode('elasticsearch')
                     ->canBeEnabled()
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->booleanNode('enabled')
```

Comment:
IIUC you removed calls like these in favor of a dynamic ` canBe(Dis|En)abled` except this one?

Reply:
I'm not sure if this call is required since I'm redefining(?) a child node. I'll remove it if it makes no difference.

Reply:
I mean the `defaultTrue` / `defaultFalse`. We will always need the `booleanNode('enabled')` calls, since otherwise we can't get to the child node (for adding `validate`).

Reply:
Seems like there's nothing we can do about this.